src/pages/ServiceDetails/ServiceInfo/ServiceInfoDestinationRules.tsx

```diff
@@ -138,24 +139,26 @@ class ServiceInfoDestinationRules extends React.Component<ServiceInfoDestination
   }
 
   rows() {
-    return (this.props.destinationRules || []).map((destinationRule, vsIdx) => ({
-      id: vsIdx,
-      name: this.overviewLink(destinationRule),
-      status: <ConfigIndicator id={vsIdx + '-config-validation'} validations={[this.validation(destinationRule)]} />,
-      trafficPolicy: destinationRule.spec.trafficPolicy ? (
-        <DetailObject name="" detail={destinationRule.spec.trafficPolicy} />
-      ) : (
-        'None'
-      ),
-      subsets:
-        destinationRule.spec.subsets && destinationRule.spec.subsets.length > 0
-          ? this.generateSubsets(destinationRule.spec.subsets)
-          : 'None',
-      host: destinationRule.spec.host ? <DetailObject name="" detail={destinationRule.spec.host} /> : undefined,
-      createdAt: <LocalTime time={destinationRule.metadata.creationTimestamp || ''} />,
-      resourceVersion: destinationRule.metadata.resourceVersion,
-      actions: this.yamlLink(destinationRule)
-    }));
+    return (this.props.destinationRules || []).map((destinationRule, vsIdx) => {
```

Comment:
This block of code shows changed in github web but I don't see the differences, perhaps it's just a format change ?

Reply:
you are right. I shouldn't be any change here. I try to remove it from the commit.